engine/src/main/java/com/datatorrent/stram/BlacklistBasedResourceRequestHandler.java

```diff
@@ -98,7 +100,19 @@ public void reissueContainerRequests(AMRMClient<ContainerRequest> amRmClient, Ma
     }
   }
 
-  public void recreateContainerRequest(Map<StreamingContainerAgent.ContainerStartRequest, MutablePair<Integer, ContainerRequest>> requestedResources, int loopCounter, ResourceRequestHandler resourceRequestor, List<ContainerRequest> removedContainerRequests)
+  @Override
+  public void addContainerRequest(Map<StreamingContainerAgent.ContainerStartRequest, MutablePair<Integer, ContainerRequest>> requestedResources, int loopCounter, List<ContainerRequest> containerRequests, StreamingContainerAgent.ContainerStartRequest csr, ContainerRequest cr)
```

Comment:
+1 to reduce the access scope.
-0 to move methods unless their implementation is modified.